CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,134 @@
+
+# Contributor Covenant Code of Conduct
```

Comment:
Since we have a COPIM-wide Code of Conduct, should we use that here? (I know it also takes elements from the Contributor Covenant - not sure how closely they overlap or whether the differences matter.)

Reply:
Good point. I'll see if we have in markdown

Reply:
I've just got a copy from Toby. Will update soon